README.md

```diff
@@ -177,6 +177,14 @@ const {
 });
 ```
 
+While this would avoid introduced a new name to the API surface there is discomfort with the shape of the output depending on the shape of the input and the risk of accidentally passing multiple arguments instead of an array - for example:
```

Comment:
Grammar tweaks:

"This would avoid introducing a new name to the API surface.  However, there is discomfort with the shape of the output, depending on the shape of the input and the risk of accidentally passing multiple arguments instead of an array.  For example:"

Reply:
... and that's why it's important to get this right.  For want of a comma, the specification could be lost... 😉